### ISSUE TITLE
Hilighted brace style or cusor enhansment for improved visibility

### ISSUE THREAD

**Issue by syonfox** [processing/p5.js-web-editor#3066]
### Increasing Access

![image](https://github.com/processing/p5.js-web-editor/assets/9345677/3eecf52a-ebba-441e-aea0-b02b470b0061)

As you can see in the images above when braces are highlighted you cannot visually tell if the cursor is on the at position 1, 2, 3,or 4.  ie `myFunction1(2i3)4`

By improving this we let people quickly visually determine the exact position of there cursor so they know where they are typing at all times.

### Feature enhancement details

Some ideas how to improve this would be to use a highlight color instead of a border box.

Another possibility is to add the feature of cursor style so that we can select the block or underscore courser. 

Let me know what you thinks

**Comment by katiejliu** [comment]
Possible example of how highlighting color would work on Brackets UI 
<img width="556" alt="Screenshot 2024-04-12 at 11 23 42 AM" src="https://github.com/processing/p5.js-web-editor/assets/78124298/38c1b06d-b7a3-44b8-98a1-84aba04f9c3f">
But this has its own visibility issue as the highlight over the line of code being grey (at least in this case) is not super visible/contrasted 

Other code editors such as VS code use border box and I personally prefer border box to the highlight color
